src/main/scala/EmrSparkPlugin.scala

```diff
@@ -49,6 +54,7 @@ object EmrSparkPlugin extends AutoPlugin {
     val sparkS3LoggingFolder = settingKey[Option[String]]("S3 folder for application's logs")
     val sparkS3JsonConfiguration = settingKey[Option[String]]("S3 location for the EMR cluster json configuration")
     val sparkAdditionalApplications = settingKey[Option[Seq[String]]]("Applications other than Spark to be deployed on the EMR cluster, these are case insensitive.")
+    val sparkEMRSteps = settingKey[Option[Seq[StepConfig]]]("Multiple steps to run once cluster is setup")
```

Comment:
Could it be `sparkEmrSteps` to align with other keys?

BTW, I'm curious about the motivation of this key, is it for "submit multiple jobs in one command" or "create `StepConfig` manually"?
It doesn't look like a convenient key for me now, since I don't want to write
```
new StepConfig()
  .withActionOnFailure(ActionOnFailure.CONTINUE)
  .withName("Spark Step")
  .withHadoopJarStep(
    new HadoopJarStepConfig()
      .withJar("command-runner.jar")
      .withArgs((Seq("spark-submit", "--deploy-mode", "cluster", "--class", mainClass, jarUrl.toString) ++ args).asJava)
  )
```
by myself in `build.sbt`. Do you have any requirement other than the settings above? For example if you want to change the name of step or the ActionOnFailure behavior?
Just curious, we can still add it anyway.